### ISSUE TITLE
Can you change the location of where models are generated when using amplify codegen models?

### ISSUE THREAD

**Issue by cmaycumber** [aws-amplify/amplify-cli#3585]
**Which Category is your question related to?**
Datastore, Amplify codegen models

Right now when I run amplify codegen models, the models are currently created in the src/models directory. Is there any way to change this?

I imagine that it would act similarly to how the normal codegen where the file location can be changed in the .graphqlconfig.yml

**Comment by aharbick** [comment]
It may be the case that this already works 🤷‍♂ 

I modified my .graphqlconfig.yml like so:

```diff
3c3
<     schemaPath: src/graphql/schema.json
---
>     schemaPath: graphql/schema.json
5c5
<       - src/graphql/**/*.js
---
>       - graphql/**/*.js
12c12,15
<         docsFilePath: src/graphql
---
>         docsFilePath: graphql
```

and ran ```amplify codegen``` on my own and it did what I think you're asking about.

BUT... beware I think I saw the automated step that is offered to you when you do ```amplify push``` overwrite my settings.

**Comment by kaustavghosh06** [comment]
@cmaycumber No, we do not allow this to be configurable at the moment but I've added a feature-request label to it for us to prioritize.

**Comment by cmaycumber** [comment]
Any update on this? I know it's not an absolute necessity but I'm currently using a monorepo without a src folder and I'd like to get rid of it once this feature finally gets introduced.

**Comment by sebastien-A256** [comment]
Same situation here. I'm using a mono repo and would appreciate a lot being able to configure where models are generated.

**Comment by danrivett** [comment]
I've had to add a workaround as follows to my `package.json` and run `yarn codegen:models` instead. Not ideal, but unless this is fixed, this will have to do.

```json
"codegen:models": "if [ -d models ]; then rm -r models; fi; amplify codegen models; if [ -d src/models ]; then mv src/models .; fi"
```
